### DISCUSSION TITLE
Webview scale is wrong

### DISCUSSION THREAD

**Discussion by Antix-Development** [microsoft/vscode-discussions#1933] [Extension Development QnA] [answered]
Hi all. I'm making a Webview based extension and the scaling is all wrong. The entire interface seems to be scaled up. I reset zoom using View > Appearance > Reset Zoom , but it's still overly large.

If I just run my HTML in a normal web browser (Chrome) it appears as intended.

Is this an easy fix? It's kind of confusing. 

It looks like this.. where the left image is the extension running in VSCode where its scaling is wrong, and on the right the same HTML running in Chrome, where it appears as expected.

![image](https://github.com/user-attachments/assets/404e8c49-e97f-48a6-b8be-b3ddf7f9dba9)

**Comment by sketchbuch** [comment, marked as answer]
Are you sure you have not increased zoom in VSCode?

**Comment by Antix-Development** [comment]
I'm pretty sure. As I mentioned I've reset the zoom using View > Appearance > Reset Zoom.

But I just tested changing the zoom level and if I zoom out making the font smaller, my extension is now the correct size.

I still have no idea whats happening and why this actually happens.

Is there any way to make VSCode respect the scaling meta in the HTML head?
